extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -116,7 +120,7 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 
 		hostnameMatch = refHostname === loc.hostname;
 
-		if (hostnameMatch && ref.indexOf('search=') > -1) {
+		if (hostnameMatch && wasWikiaSearchUsed(ref)) {
```

Comment:
what do you think about creating a separate function for checking whether hostname matches wikiDomainsRegex ?